<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KalignPairwiseAlignmentOptionsWidget</class>
 <widget class="QWidget" name="KalignPairwiseAlignmentOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="gapOpenLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="gapOpenLabel">
       <property name="text">
        <string>Gap open penalty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gapOpen">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="gapExtdLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="gapExtdLabel">
       <property name="text">
        <string>Gap extension penalty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gapExtd">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="gapTermLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="gapTermLabel">
       <property name="text">
        <string>Terminal gap extension penalty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gapTerm">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
